packages/x-scheduler-headless/src/calendar-grid/day-event/CalendarGridDayEvent.tsx

```diff
@@ -8,7 +8,7 @@ import { useButton } from '../../base-ui-copy/utils/useButton';
 import { useRenderElement } from '../../base-ui-copy/utils/useRenderElement';
 import { BaseUIComponentProps } from '../../base-ui-copy/utils/types';
 import { useEvent } from '../../utils/useEvent';
-import { CalendarEventId, SchedulerValidDate } from '../../models';
+import { CalendarEvent, CalendarEventId, SchedulerValidDate } from '../../models';
 import { useAdapter, diffIn } from '../../use-adapter/useAdapter';
```

Comment:
```suggestion
import { useAdapter, diffIn } from '../../use-adapter';
```
We should update all the imports like you did in `useDayCellDropTarget.ts`